build/jenkins/Jenkinsfile.development

```diff
@@ -11,6 +11,7 @@ pipeline {
   }
   environment {
     BUILD_DATE = sh(returnStdout: true, script: "date -u +'%Y-%m-%dT%H:%M:%S%z'").trim()
+    TAG = sh(returnStdout: true, script: "make project-get-tag BUILD_DATE=${BUILD_DATE}").trim()
```

Comment:
After integrating with the latest Make DevOps scripts this target will need to be renamed to `project-get-build-tag`